packages/orchestrator-ui-components/src/rtk/endpoints/forms.ts

```diff
@@ -0,0 +1,64 @@
+import { BaseQueryTypes, orchestratorApi } from '@/rtk';
+
+const PROCESS_ENDPOINT = 'processes';
+const RESUME_ENDPOINT = 'resume';
+const FORMS_ENDPOINT = 'surf/forms/'; // It is still being used by example-wfo-uo
+
+const formsApi = orchestratorApi.injectEndpoints({
+    endpoints: (build) => ({
+        startProcess: build.mutation<
+            unknown,
```

Comment:
This 'unknow' type is the return value right? Are you sure it's unknown? If there's not return value you can say void I think

Reply:
`    startProcess = (
        workflowName: string,
        processInput: object,
    ): Promise<unknown> => {
        return this.postPutJson<unknown>(
            `${PROCESS_ENDPOINT}${workflowName}`,
            processInput,
            'post',
            false,
            true,
        );
    };`
    
    This was the axios version, that is why I used unknown